frontend/src/pages/NewRunV2.tsx

```diff
@@ -271,31 +291,76 @@ function NewRunV2(props: NewRunV2Props) {
       resource_references: apiResourceRefFromRun ? apiResourceRefFromRun : references,
       service_account: serviceAccount,
     };
+
+    let newRecurringRun: ApiJob = Object.assign(
+      newRun,
+      isRecurringRun
+        ? {
+            enabled: true,
+            max_concurrency: maxConcurrentRuns || '1',
+            no_catchup: !catchup,
+            trigger: trigger,
+          }
+        : {
+            enabled: false,
+            max_concurrency: undefined,
+            no_catchup: undefined,
+            trigger: undefined,
+          },
+    );
     setIsStartingNewRun(true);
 
-    newRunMutation.mutate(newRun, {
-      onSuccess: data => {
-        setIsStartingNewRun(false);
-        if (data.run?.id) {
-          props.history.push(RoutePage.RUN_DETAILS.replace(':' + RouteParams.runId, data.run.id));
-        }
-        props.history.push(RoutePage.RUNS);
-
-        props.updateSnackbar({
-          message: `Successfully started new Run: ${data.run?.name}`,
-          open: true,
-        });
-      },
-      onError: async error => {
-        const errorMessage = await errorToMessage(error);
-        props.updateDialog({
-          buttons: [{ text: 'Dismiss' }],
-          onClose: () => setIsStartingNewRun(false),
-          content: errorMessage,
-          title: 'Run creation failed',
-        });
-      },
-    });
+    const runCreation = () =>
+      newRunMutation.mutate(newRun, {
+        onSuccess: data => {
+          setIsStartingNewRun(false);
+          if (data.run?.id) {
+            props.history.push(RoutePage.RUN_DETAILS.replace(':' + RouteParams.runId, data.run.id));
+          }
+          props.history.push(RoutePage.RUNS);
+
+          props.updateSnackbar({
+            message: `Successfully started new Run: ${data.run?.name}`,
+            open: true,
+          });
+        },
+        onError: async error => {
+          const errorMessage = await errorToMessage(error);
+          props.updateDialog({
+            buttons: [{ text: 'Dismiss' }],
+            onClose: () => setIsStartingNewRun(false),
+            content: errorMessage,
+            title: 'Run creation failed',
+          });
+        },
+      });
+
+    const recurringRunCreation = () =>
+      newRecurringRunMutation.mutate(newRecurringRun, {
+        onSuccess: data => {
+          setIsStartingNewRun(false);
+          if (data.id) {
+            props.history.push(RoutePage.RUN_DETAILS.replace(':' + RouteParams.runId, data.id));
```

Comment:
I don't understand this part: How to navigate to a run detail page when it is recurring run?

Reply:
It should be navigate to a recurring run detail page instead.